mmv1/products/apigee/terraform.yaml

```diff
@@ -298,6 +298,31 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       delete_minutes: 30
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       custom_import: templates/terraform/custom_import/apigee_endpoint_attachment.go.erb
+  NatAddress: !ruby/object:Overrides::Terraform::ResourceOverride
+    autogen_async: true
+    import_format: ["{{instance_id}}/natAddresses/{{name}}", "{{instance_id}}/{{name}}"]
+    delete_url: '{{instance_id}}/natAddresses/{{name}}'
+    skip_sweeper: true
+    examples:
+    - !ruby/object:Provider::Terraform::Examples
+      name: "apigee_nat_address_basic"
+      skip_test: true
+    - !ruby/object:Provider::Terraform::Examples
+      # This is a more verbose version of the above that creates all
+      # the resources needed for the acceptance test.
+      name: "apigee_nat_address_basic_test"
```

Comment:
Hi @danistrebel thanks for your PR!

I just took a quick look and saw that these name values need an update. The value of `name` needs to match the name of the `mmv1/templates/terraform/examples/*.tf.erb` files you've added, otherwise magic-modules experiences a missing files error when building the provider code. Could you please push an update to these values?